src/run_sign.py

```diff
@@ -8,19 +8,22 @@
 
 import argparse
 import logging
-import os
 import sys
+from pathlib import Path
 
-from manifests.build_manifest import BuildManifest
-from sign_workflow.signer import Signer
+from sign_workflow.sign_existing_artifacts import SignExistingArtifacts
+from sign_workflow.sign_with_manifest import SignWithManifest
 from system import console
 
+ACCEPTED_SIGNATURE_FILE_TYPES = [".sig", ".asc"]
+
 
 def main():
     parser = argparse.ArgumentParser(description="Sign artifacts")
-    parser.add_argument("manifest", type=argparse.FileType("r"), help="Path to local manifest file.")
+    parser.add_argument("target", type=Path, help="Path to local manifest file or artifact directory")
     parser.add_argument("--component", nargs="?", help="Component name")
     parser.add_argument("--type", nargs="?", help="Artifact type")
+    parser.add_argument("--sigType", nargs="?", help="Type of Signature file", default=".asc")
```

Comment:
Arguments shouldn’t be camelcase, —sigtype or —signature-type.
Lowercase signature in the description.